.github/workflows/ci.yml

```diff
@@ -19,41 +19,66 @@ jobs:
       uses: actions/setup-node@v3
       with:
         node-version: ${{ matrix.node-version }}
+        cache: 'npm'
+        cache-dependency-path: 'packages/**/package-lock.json'
```

Comment:
Caching the npm dependencies should give us a nice speedup for CI runs when package-lock doesn't change.